### DISCUSSION TITLE
`Next-ValidEnv` for Typesafe environment variables

### DISCUSSION THREAD

**Discussion by JacobADevore** [vercel/next.js#45175] [Ideas]
### Describe the feature you'd like to request

Create an example repo to show how to use Next-ValidEnv to bring safety to your environment variables.

https://twitter.com/JacobADevore/status/1614832929036972032

### Describe the solution you'd like

Take the with-package example shown in Next-ValidEnv and create a 'with-typesafe-env' example in Next.js

### Describe alternatives you've considered

Remain as is...

**Comment by balazsorban44** [comment]
Hi, check out the contributing docs on how to add an example: https://github.com/vercel/next.js/blob/canary/contributing/examples/adding-examples.md
